src/Xamarin.Android.Build.Tasks/Mono.Android/ServiceAttribute.Partial.cs

```diff
@@ -31,7 +33,16 @@ partial class ServiceAttribute {
 			  "exported",
 			  self          => self.Exported,
 			  (self, value) => self.Exported  = (bool) value
-			}, {
+			},
+			#if ANDROID_29
```

Comment:
This code shouldn't be Conditional.  (It *can*, be, but *shouldn't* be, because if it is removed, things won't work as expected at packaging time, and tracking that down will be annoying.)

Reply:
Fixed